lib/toy/module.rb

```diff
@@ -1,4 +1,5 @@
 require "toy/behaviours"
+require "toy/object"
```

Comment:
Perhaps unrelated, but while reviewing this I noticed that, strictly speaking, we should also `require "toy/class"`, because `Module.class` returns `Class`.